auth_backends/backends.py

```diff
@@ -290,6 +292,16 @@ def auth_complete_params(self, state=None):
         params['token_type'] = 'jwt'
         return params
 
+    def auth_complete(self, *args, **kwargs):
```

Comment:
Can you explain why this is needed?

Reply:
Explain it in the code, or as a github comment?  It is needed during Insights auth to initiate the fetching of the "allowed" courses from the courses list API.

https://github.com/edx/edx-analytics-dashboard/pull/875/files#diff-d348b2b93c2b58adcb832a4e73324fc0R221

Reply:
Ok. I understand now.